blog/content/post/2019-05-15-building-bareflank.md

```diff
@@ -0,0 +1,345 @@
+---
+post_title: Building and running Bareflank
+author: krystian.hebel
+layout: post
+published: true
+date: 2019-05-15
+
+tags:
+    - hypervisor
+    - virtualization
+    - bare-metal
+    - VMX
+categories:
+    - Firmware
+
+---
+
+In this second post of a series, we will build and start our first hypervisor.
+It won't do much just yet, but it is good to get known with its build system.
+We will be using [Bareflank](https://github.com/Bareflank/hypervisor).
+
+\################################################################################
+TBD: link first post here
```

Comment:
@krystian-hebel link should look like: `https://blog.3mdeb.com/2019/2019-04-30-5-terms-every-hypervisor-developer-should-know/`